source/frontend/src/components/contact/ContactManagerView/ContactManagerView.tsx

```diff
@@ -83,9 +83,8 @@ const ContactManagerView = ({
   const changeFilter = useCallback(
     (filter: IContactFilter) => {
       setFilter(filter);
-      setCurrentPage(0);
```

Comment:
due to my updates to setfilter, the page is always set to 0 internally in the useSearch hook anyways. This no longer fires two search requests.